tests/fail/tree-borrows/propagation/unknown-size-protected.stderr

```diff
@@ -0,0 +1,59 @@
+──────────────────────────────────────────────────────────────────────
+Warning: this tree is indicative only. Some tags may have been hidden.
+0..  8
+| Act|    └─┬──<TAG=[root]>
+|----|      └─┬──<TAG=data>
+|----|        ├─┬──<TAG=x>
+|----|        │ └────<TAG=arg:x> Strongly protected
+|----|        └────<TAG=y,arg:y>
+──────────────────────────────────────────────────────────────────────
+──────────────────────────────────────────────────────────────────────
+Warning: this tree is indicative only. Some tags may have been hidden.
+0..  7..  8
+| Act| Act|    └─┬──<TAG=[root]>
+|----| Act|      └─┬──<TAG=data>
+|----|?Re*|        ├─┬──<TAG=x>
+|----|?Dis|        │ └────<TAG=arg:x> Strongly protected
+|----| Act|        └────<TAG=y,arg:y>
+──────────────────────────────────────────────────────────────────────
+──────────────────────────────────────────────────────────────────────
+Warning: this tree is indicative only. Some tags may have been hidden.
+0..  8
+| Act|    └─┬──<TAG=[root]>
+|----|      └─┬──<TAG=data>
+|----|        ├─┬──<TAG=u>
+|----|        │ └────<TAG=arg:u> Strongly protected
+|----|        └────<TAG=v,arg:v>
+──────────────────────────────────────────────────────────────────────
+──────────────────────────────────────────────────────────────────────
+Warning: this tree is indicative only. Some tags may have been hidden.
+0..  7..  8
+| Act| Act|    └─┬──<TAG=[root]>
+|----| Re*|      └─┬──<TAG=data>
+|----| Re*|        ├─┬──<TAG=u>
+|----| Re*|        │ └────<TAG=arg:u> Strongly protected
+|----|?Re*|        └────<TAG=v,arg:v>
+──────────────────────────────────────────────────────────────────────
+error: Undefined Behavior: Protector triggered on <TAG> whose permissions were updated from Permission(Reserved { ty_is_freeze: false }) to Permission(Disabled). Caused by a foreign Write through <TAG>.
+  --> $DIR/../../../utils/tb/mod.rs:LL:CC
+   |
+LL |             *ptr.add($offset) = 0;
+   |             ^^^^^^^^^^^^^^^^^^^^^ Protector triggered on <TAG> whose permissions were updated from Permission(Reserved { ty_is_freeze: false }) to Permission(Disabled). Caused by a foreign Write through <TAG>.
```

Comment:
What about

"write access through \<TAG> is forbidden because it is a foreign tag for \<TAG>, which would hence change from reserved to disabled, but \<TAG> is protected"